### PR TITLE
added topic modeling walkthrough

### PR DESCRIPTION
**Pull request by etano**

### CODE REVIEW

**Comment by alexdelin** [comment]
@etano I would probably move the actual content to s3 or something and just include a wget command to download it

**Comment by etano** [comment]
that doesn't sound like the @alexdelin I know, but ok.

**Comment by alexdelin** [comment]
lol
or maybe just compress it and add it with git LFS?
What do you think is better?

**Comment by etano** [comment]
ok, added a fetch.sh

File: ml_service/1_topic_modeling/fetch.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+DATADIR=$(dirname "$0")
+DATASET=20news-18828
+
+aws s3 sync s3://squirro-datasets/walkthroughs/$DATASET $DATADIR --profile squirro-internal
```
Comment (alexdelin) [inline ml_service/1_topic_modeling/fetch.sh]: This repo is intended mainly for partners, so I wouldn't count on anyone being able to access our squirro-internal AWS buckets. 
I'm going to put together a separate PR that re-works this across the board to make it easier

File: ml_service/1_topic_modeling/20 newsgroups.ipynb [hunk 2]
```diff
@@ -0,0 +1,692 @@
...
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Embedding step\n",
+    "\n",
+    "Right before classification, we have to convert our list of tokenized words into numbers. This is done via an `embedder` step. Squirro comes shipped with some pre-trained embeddings, but for this case, we'll make our own doc2vec embeddings."
```
[the remark this answers is not in the document]
Comment (etano) [reply ml_service/1_topic_modeling/20 newsgroups.ipynb]: no, it's tfidf

File: ml_service/1_topic_modeling/fetch.sh (hunk 1 above)
Comment (etano) [reply ml_service/1_topic_modeling/fetch.sh]: across the board?
